src/s3fs.cpp

```diff
@@ -2946,6 +2946,20 @@ static int s3fs_release(const char* _path, struct fuse_file_info* fi)
             return result;
         }
 
+        // [NOTE]
+        // There are cases when s3fs_flush is not called and s3fs_release is called.
+        // (There have been reported cases where it is not called when exported as NFS.)
+        // Therefore, Flush() is called here to try to upload the data.
+        // Flush() will only perform an upload if the file has been updated.
+        //
+        if(ent->IsModified()){
+            if(0 != (result = ent->Flush(static_cast<int>(fi->fh), AutoLock::NONE, false))){
```

Comment:
before uploading the content, the  NoTruncateFlag is cleared at  2915， is  this ok？